addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.xml

```diff
@@ -2,54 +2,41 @@
+            />
+            <t t-else="" t-call="mail.DiscussSidebarCallParticipants.list"/>
         </div>
     </t>
 
-    <t t-name="mail.DiscussSidebarCallParticipants.participant">
-        <t t-set="channelMember" t-value="session.channelMember"/>
-        <div class="d-flex text-reset overflow-hidden align-items-center" t-att-class="{ 'justify-content-center bg-inherit': compact, 'me-2': !compact and props.compact === undefined }">
-            <div class="bg-inherit position-relative d-flex flex-shrink-0" style="width:24px;height:24px">
-                <img class="o-mail-DiscussSidebarCallParticipants-avatar w-100 h-100 rounded-circle o_object_fit_cover" t-att-src="channelMember.persona.avatarUrl" t-att-class="{'o-isTalking': !session.isMute and session.isTalking}" alt="Participant avatar"/>
-            </div>
-            <span t-if="!compact" class="o-mail-DiscussSidebarCallParticipants-name mx-1 text-truncate fw-bolder smaller user-select-none" t-att-title="channelMember.persona.name" t-att-class="{ 'o-isTalking': !session.isMute and session.isTalking }">
-                <t t-esc="channelMember.persona.name"/>
-            </span>
-            <div t-if="!compact" class="flex-grow-1"/>
-            <div class="o-mail-DiscussSidebarCallParticipants-status" t-att-class="{ 'position-absolute bg-inherit p-0 rounded-circle o-compact d-flex smaller start-0 text-start': compact, 'ms-1 d-flex align-items-center justify-content-center': !compact }">
-                <t t-if="compact">
-                    <span t-if="session.shortStatus" t-att-class="{
-                        [`fa ${callActionsRegistry.get('mute').icon}`]: session.shortStatus === 'mute',
-                        [`fa ${callActionsRegistry.get('deafen').icon}`]: session.shortStatus === 'deafen',
-                        'o-live bg-danger o-text-white rounded': session.shortStatus === 'live',
-                    }"><t t-if="session.shortStatus === 'live'">LIVE</t></span>
-                </t>
-                <t t-else="">
-                    <span t-if="session.isMute" class="p-1 fa" t-att-class="callActionsRegistry.get('mute').icon"/>
-                    <span t-if="session.isDeaf" class="p-1 fa" t-att-class="callActionsRegistry.get('deafen').icon"/>
-                    <span t-if="session.isScreenSharingOn" class="o-live bg-danger o-text-white rounded">LIVE</span>
-                </t>
+    <t t-name="mail.DiscussSidebarCallParticipants.list">
+        <div class="d-flex flex-column gap-1">
+            <div class="d-flex align-items-center" t-foreach="sessions" t-as="session" t-key="session.localId">
+                <t t-set="persona" t-value="session.channelMember.persona"/>
+                <img t-att-src="persona.avatarUrl" class="rounded-circle" t-att-title="persona.name" t-att-class="{'o-mail-DiscussSidebarCallParticipants-avatar o-isTalking': persona.currentRtcSession.isActuallyTalking}" style="width:24px;height:24px"/>
```

Comment:
Probably best to pick `session.persona` rather than `session.channelMember.persona`, as the links `session <-> channelMember <-> session` may be lacking, and the alias handles it internally through `?.`.

Lack of data may happen and we should embrace it :)